src/sentry/tasks/post_process.py

```diff
@@ -600,12 +600,16 @@ def process_rules(job: PostProcessJob) -> None:
     return
 
 
+@instrumented_task(name="sentry.tasks.post_process.process_code_mappings")
```

Comment:
I think it might be fine with not tracing this. I can remove this if you agree with me.

Reply:
This converts this function into a task. I don't recommend instrumenting this way, since it looks like the function is meant to be called async. I don't mind whether you want to instrument or not though.